apps/client/src/App.jsx

```diff
@@ -20,9 +23,12 @@ function App() {
         <Route path="/wishlist" element={<WishList />} />
         <Route path="/login" element={<LoginForm />} />
         <Route path="/signup" element={<SignupForm />} />
-        <Route path="/fashion" element={<Fashion />} />
-        <Route path="/food" element={<Food />} />
-        <Route path="/home" element={<Home />} />
+
+        <Route path="/Fashion" element={<Fashion />} /> 
+        <Route path="/Food" element={<Food />} /> 
+        <Route path="/Home" element={<Home />} /> 
+        <Route path="/admin" element={<AdminCouponList/>} />
+
         <Route path="/beauty" element={<Beauty />} />
```

Comment:
@noagedo Please sync with @yuvalbenshitrit 
she change all those paths to be with lowercases